app/src/test/java/com/github/takahirom/roborazzi/sample/ComposeTest.kt

```diff
@@ -33,4 +54,26 @@ class ComposeTest {
         .performClick()
     }
   }
+
+  @Test
+  @Config(
+    sdk = [30],
+    qualifiers = "w221dp-h221dp-small-notlong-round-watch-xhdpi-keyshidden-nonav"
```

Comment:
👍

Reply:
We may want to provide these as "const val" for the same devices that are provided in Android Studio previews.